website/docs/kubernetes/creating-a-kube-cluster.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 11
+title: Creating a Kubernetes cluster
+description: Creating a Kubernetes cluster
+keywords: [podman desktop, podman, creating, cluster, kubernetes]
+tags: [creating-kubernetes-cluster, extentions]
+---
+
+# Creating a Kubernetes cluster using extensions
```

Comment:
This Kubernetes category does not cover any section for setting up or creating a Kubernetes cluster. Though we have covered this part in the extension sections below, but users might look out for the same details here as well. To make the content accessible and cohesive, added a reference section wherein all the procedural sections that they need to follow are covered. so, they can navigate and perform the required steps from the Kubernetes section as well.